composer.json

```diff
@@ -13,11 +13,11 @@
         }
     ],
     "require": {
-        "php": ">=5.2.4"
+        "php": ">=5.6"
     },
     "require-dev": {
-        "phpunit/phpunit": "~3.7|~4.0|~5.0",
-        "friendsofphp/php-cs-fixer": "~1.11"
+        "friendsofphp/php-cs-fixer": "~2.19.3",
+        "yoast/phpunit-polyfills": "^2.0"
```

Comment:
My biggest question here is whether we need to keep such old PHP version support, a such, such old PHPUnit version support... and therefore whether we actually need the polyfills at all

Reply:
For consuming projects like WP-CLI, PHP 5.6 is still required this year. After that, 7.0 or 7.2.

But even then, there is no single PHPUnit version that covers all PHP versions and is fully supported, see https://phpunit.de/supported-versions.html.

Plus, the polyfills provide _forward_ compatibility, so that you can always use the latest PHPUnit features, even if you still need to use PHPUnit 8 for older PHP versions. Thus, this package is still very useful even if the project requires more modern PHP versions.